src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskSearchControllerTest.java

```diff
@@ -94,9 +94,9 @@ public void should_return_a_200_with_search_results_based_on_jurisdiction_and_lo
 
         result.then().assertThat()
             .statusCode(HttpStatus.OK.value())
-            .body("tasks.state", everyItem(either(is("unassigned")).or(is("assigned"))))
-            .body("tasks.case_data.reference", hasItem(task.get("ccdId")))
-            .body("tasks.case_data.location.id", everyItem(equalTo("17595")));
+            .body("tasks.task_state", everyItem(either(is("unassigned")).or(is("assigned"))))
+            .body("tasks.case_id", hasItem(task.get("ccdId")))
+            .body("tasks.location", everyItem(equalTo("17595")));
     }
 
```

Comment:
do we need a functional test to cover the get Acceptance criteria? i.e. GET /task/{task-id} 
To also show/assert all the standard attributes